homeassistant/components/climate/__init__.py

```diff
@@ -48,6 +48,9 @@
 STATE_HIGH_DEMAND = 'high_demand'
 STATE_HEAT_PUMP = 'heat_pump'
 STATE_GAS = 'gas'
+STATE_MANUAL = 'manual'
```

Comment:
I just realized that this is not correct. Things like manual and temporary are hold modes and should be returned from that property.

Reply:
Is there a description what hold modes are? (It's not intuitive for me, non native speaker) I just implemented it as hold modes but there is no way to set them from the frontend?